### DISCUSSION TITLE
I havent got access to Github Education pack

### DISCUSSION THREAD

**Discussion by bhatganeshdarshan** [community/community#147590] [GitHub Education] [answered]
### Hi everyone,

my application is showing accepted , but i havent got access to Github student developer pack yet . Here is the image ..
![image](https://github.com/user-attachments/assets/314bbb6b-f6ef-44d9-bdb6-83fbeca14c6f)

**Comment by tuves** [comment, marked as answer]
Hi @bhatganeshdarshan, you'll usually receive your benefits after 3 days of this approval status. Thank you for being patient!
